asm/caxpy.go

```diff
@@ -1,36 +0,0 @@
-// Generated code do not edit. Run `go generate`.
```

Comment:
Rebase artifact.  Moved to stubs_noasm.go in #7ba700f45f695c4bdc63490f7f69b66922e6f5c7